src/cljs/main/broadfcui/page/workspace/monitor/workflow_details.cljs

```diff
@@ -177,94 +177,112 @@
-         (links/create-external {:href (str moncommon/google-cloud-context (:bucketName props) "/" (:submission-id props)
-                                            "/" workflow-name "/" (:workflowId props) "/call-" call-name "/")}
-           (:label props)))]
-      (links/create-internal {:onClick #(swap! state update :expanded not)}
-        (if (:expanded @state) "Hide" "Show"))
-      (when (:expanded @state)
-        (map-indexed
-         (fn [index data]
-           [:div {:style {:padding "0.5em 0 0 0.5em"}}
-            [:div {:style {:paddingBottom "0.25em"}} (str "Call #" (inc index) ":")]
-            [:div {:style {:paddingLeft "0.5em"}}
-             (create-field "Operation"
-                           ;; Note: using [:a ...] instead of style/create-link to be consistent with GCSFilePreviewLink
-                           [:a {:href "javascript:;"
-                                :onClick (fn [_]
-                                           (swap! state assoc
-                                                  :show-operation-dialog? true
-                                                  :operation-dialog-props {:workspace-id (:workspace-id props)
-                                                                           :job-id (data "jobId")
-                                                                           :dismiss #(swap! state dissoc :show-operation-dialog?)}))}
-                            (data "jobId")])
-             (let [status (data "executionStatus")]
-               (create-field "Status" (moncommon/icon-for-call-status status) status))
-             (when (= (get-in data ["callCaching" "effectiveCallCachingMode"]) "ReadAndWriteCache")
-               (create-field "Cache Result" (moncommon/format-call-cache (get (data "callCaching") "hit"))))
-             (create-field "Started" (moncommon/render-date (data "start")))
-             ;(utils/cljslog data)
-             (create-field "Ended" (moncommon/render-date (data "end")))
-             [IODetail {:label "Inputs" :data (data "inputs") :call-detail? true}]
-             [IODetail {:label "Outputs" :data (data "outputs") :call-detail? true}]
-             (create-field "stdout" (display-value (data "stdout") (last (string/split (data "stdout") #"/"))))
-             (create-field "stderr" (display-value (data "stderr") (last (string/split (data "stderr") #"/"))))
-             (backend-logs data)
-             (when-let [failures (data "failures")]
-               [Failures {:data failures}])]])
-         (:data props)))])})
+     (let [subWorkflowId (:subWorkflowId props)]
+       [:span {}
+        (links/create-internal {:onClick #(swap! state update :expanded not)}
+                               (if (:expanded @state) "Hide" "Show"))
+        [:div {}
+         (when (:expanded @state)
+           [WorkflowDetails (merge props {:workflow-id subWorkflowId})])]]))})
 
+(defn- render-gcs-path [components]
```

Comment:
Example Input: 
```[fc-13226cd9-8f18-4969-a7fd-95dc367ee5e1,
33c68fba-3cef-40a2-9e5e-8ecffe6d8750,
scatterHC_samples,
554acfd2-769e-4a60-b5d2-3d79bb3cb904]``

Output: https://accounts.google.com/AccountChooser?continue=https://console.cloud.google.com/storage/browser/fc-13226cd9-8f18-4969-a7fd-95dc367ee5e1/33c68fba-3cef-40a2-9e5e-8ecffe6d8750/scatterHC_samples/554acfd2-769e-4a60-b5d2-3d79bb3cb904/

Reply:
This allows me to recursively build the GCS paths.  After https://github.com/broadinstitute/cromwell/issues/3623 we can discard this and use the `workflowRoot` value in the metadata directly.